vignettes/bambu.Rmd

```diff
@@ -156,46 +154,83 @@ annotation, and then quantify the transcript expression from these extended
+ 
+### Modulating the sensitivity of discovery (pre and post analysis)
+
+When doing transcript discovery there is a balance between sensitivity (the number of real novel transcripts that are detected) and the precision (how many of the novel transcripts are real). To control this balance, *bambu* uses the novel discovery rate (NDR) as the main parameter. The NDR threshold approximates the proportion of novel candidates output by bambu, relative to the number of known transcripts it found, i.e., an NDR of 0.1 would mean that 10% of all transcripts passing the threshold are classified as novel. 
+
+If you are using a genome where you expect a high amount of novel transcripts, a higher NDR is recommended so that these novel transcripts are not missed. Conversely if you are using a well annotated genome, we recommend a lower NDR threshold to reduce the presence of false positives. By default the NDR threshold is automatically chosen for the user based on predicted level of annotation completeness when compared to the default model trained on human reference annotations (Hg38). For more information see [Training a model on another species/dataset and applying it](#Training-a-model-on-another-speciesdataset-and-applying-it)
+
+To manually select an NDR value, use the NDR argument in *bambu*:
+
+```{r, results = "hide"}
+se.NDR_0.3 <- bambu(reads = bamFiles, annotations = HsChr22BambuAnnotation,
+    genome = genomeSequence, NDR = 0.3)
+```
+Alternatively transcript discovery can be run without thresholds, producing a GRangesList annotation object with all transcripts scored with its NDR score. Note that this means turning quant = FALSE in running *bambu* (refer to [“Transcript discovery only”](#transcript-discovery-only-no-quantification) section). The annotations can be filtered by their NDR score (see example below), read count and gene read proportion between the discovery and quantification steps or used for other types of analysis. 
+
+```{r, results = 'hide'}
+newAnnotations <- bambu(reads = bamFiles, annotations = HsChr22BambuAnnotation,
+    genome = genomeSequence, NDR = 1, quant = FALSE)
+annotations.filtered <- newAnnotations[(!is.na(mcols(newAnnotations)$txNDR) & mcols(newAnnotations)$txNDR<0.1) | is.na(mcols(newAnnotations)$txNDR)]
+se.NDR_1 <- bambu(reads = bamFiles, annotations = annotations.filtered, genome = genomeSequence, NDR = 1, discovery = FALSE)
+```
+Additionally there are other thresholds that advanced users can access through opt.discovery when running *bambu* (see arguments).
 
 ### Visualise results {#visualise-results}               
 *bambu* provides functions to visualise and explore the results. When multiple
 samples are used, we can visualise the correlation and clustering of all
 samples with a heatmap:
 ```{r, fig.width = 8, fig.height = 6}
 library(ggplot2)
-plotBambu(se, type = "heatmap")
+plotBambu(se.multiSample, type = "heatmap")
```

Comment:
Error in loadNamespace(x) : there is no package called ‘ComplexHeatmap’

Reply:
can you try to install complexheatmap and try again? it runs through on my side without errors